modules/fedimint-ln-tests/tests/tests.rs

```diff
@@ -653,6 +665,69 @@ mod fedimint_migration_tests {
         .await;
     }
 
+    async fn create_client_db_with_v0_data(mut dbtx: DatabaseTransaction<'_>) {
+        dbtx.insert_new_entry(&DatabaseVersionKey, &DatabaseVersion(0))
+            .await;
+
+        // Generate fake private/public key
+        let (_, pk) = secp256k1::generate_keypair(&mut OsRng);
+        dbtx.insert_new_entry(
+            &fedimint_ln_client::db::LightningGatewayKey,
+            &LightningGatewayRegistration {
```

Comment:
style: Kind of a long and ugly fixture. Maybe we could move it out of the way (into a constant or some util function).